clients/python-wrapper/lakefs/__main__.py

```diff
@@ -155,16 +155,12 @@ def run_binary(binary_path: str, args: list[str]):
     '''
     Run the binary with the provided arguments.
     '''
-    if not binary_path:
-        raise RuntimeError("binary not found")
-    binary_name = os.path.basename(binary_path)
+    print(f'running {binary_path}...')
     try:
-        print(f'running {binary_path}...')
-        proc = subprocess.run(
-            [binary_path] + args, check=False, env=os.environ)
```

Comment:
`check=False` is the default. Same with `env`.

Reply:
[Looks like this is because of `pylint`](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/subprocess-run-check.html).